src/display/annotation_layer.js

```diff
@@ -96,14 +97,17 @@ AnnotationElementFactory.prototype =
  * @alias AnnotationElement
  */
 var AnnotationElement = (function AnnotationElementClosure() {
-  function AnnotationElement(parameters) {
+  function AnnotationElement(parameters, isRenderable) {
+    this.isRenderable = isRenderable;
```

Comment:
> This can be even easier, just like in Snuffleupagus' solution:

Please note that the diff I linked to wasn't actually tested, so it seems fine as-is.
However, one small suggestion I'd make is to change this line to `this.isRenderable = isRenderable || false`, so that it's not left undefined if the parameter is omitted.